app/lib/discourse_automation/scripts/close_topic.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+DiscourseAutomation::Scriptable::CLOSE_TOPIC = 'close_topic'
+
+DiscourseAutomation::Scriptable.add(DiscourseAutomation::Scriptable::CLOSE_TOPIC) do
+  field :topic_to_be_closed, component: :text, required: true
```

Comment:
I think you can use topic here, it's explicit enough. We are in close topic script, we have a topic variable, sounds clear enough.